modules/client/lobby/lobby.tsx

```diff
@@ -3,13 +3,16 @@ import {connectSocket, currentSocket} from "../core/services/socket.io.client.se
 import './css/lobby.css'
 import * as GameSettings from '../../shared/game/config/game.shared.game.config'
 
-export const LobbyPage = ({user, setPage}) => {
+export const LobbyPage = ({user, setPage, setRoomName}) => {
   const [rooms, setRooms] = React.useState([])
   const [newRoomName, setNewRoomName] = React.useState('')
   const [error, setError] = React.useState()
   const roomNameRef = React.useRef(null)
   React.useEffect(() => {
-    connectSocket(user)
+    console.log('requesting rooms')
+    if (!currentSocket.socket) {
```

Comment:
make this change to ensure the client only has one socket. when we have 2 sockets (the angular and react socket), the server thinks we are 2 clients